tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/Mono.Android.NET-Tests.csproj

```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(DotNetAndroidTargetFramework)</TargetFramework>
-    <SupportedOSPlatformVersion>21</SupportedOSPlatformVersion>
+    <SupportedOSPlatformVersion>23</SupportedOSPlatformVersion>
```

Comment:
If we bump this I think we'll need to also adjust the nightly test matrix values:
https://github.com/xamarin/xamarin-android/blob/de9d6c87b6ccab5c7784bdafa7d03c65d21814da/build-tools/automation/azure-pipelines-nightly.yaml#L82

I know there was also some discussion about bumping our min supported version to 23 in .NET 9, has that decision officially been made yet?